src/test/run-pass/spawn2.rs

```diff
@@ -12,7 +12,7 @@ use std::thread;
 
 pub fn main() {
     let t = thread::spawn(move|| child((10, 20, 30, 40, 50, 60, 70, 80, 90)) );
-    t.join().ok().unwrap();
+    t.join().ok().unwrap(); // forget Err value, since it doesn't implement Debug
```

Comment:
`join()` returns a `Result<T, Box<Any + Send + 'static>>`

there are lots of other cases where the result is ignored since `Debug` cannot be implemented generically to return "thread panicked" on `Err`